PEP 582: Add more explanation based on feedback (#2983)
python/peps: pep-0582.rst

is still far easier than learning the details of virtual environments in various
platforms.

A major point for this PEP is that it is not trying to replace virtual environments.
If one needs all the features of virtual environments, they should use proper virtual

> environments (example, created using venv module).

Reviewer: ```suggestion
environments (for example, created using the :mod:`venv` module).
```

Fix grammar and mark up & link `venv` accordingly.

check out the source code, a tool like ``pip`` can be used to install the
required dependencies directly into this directory.

> But, this does not enable all features of virtual environments in the similar

Reviewer: ```suggestion
But, this does not enable all features of virtual environments in a similar
```

Fix grammar

> fashion. For example, if the project has multiple scripts, or helper scripts
> in a different directory to build the project, instead of using
> ``__pypackages__`` using a normal virtual environment should be the preferred
> method.

Reviewer: ```suggestion
fashion. For example, if the project has multiple scripts, or helper scripts
in a different directory to build the project, a normal virtual environment
should be preferred over ``__pypackages__``.
```

Fix grammar/phrasing/clarity

``/usr/share/myproject/`` directory, it will be used. Any potential
``__pypackages__`` directory in ``/tmp`` will be ignored.

> This also means we will not scan any parent directory while executing any
> script. If we want to execute scripts inside of ``~/bin/`` directory, then
> the ``__pypackages__`` directory must be inside of ``~/bin/`` directory.

Reviewer: ```suggestion
This also means we will not scan any parent directory while executing scripts.
If we want to execute scripts inside of the ``~/bin/`` directory, then
the ``__pypackages__`` directory must be inside of the ``~/bin/`` directory.
```

Fix a couple grammar errors (missing definite articles) and clarify phrasing/remove repeated word.

> This also means we will not scan any directory above to while executing any
> script. If we want to execute scripts inside of ``~/bin/`` directory, then
> the ``__pypackages__`` directory must be inside of ``~/bin/`` directory.

Reviewer 2: ```suggestion
This also means we will not scan any parent directory while executing any
script. If we want to execute scripts inside of ``~/bin/`` directory, then
the ``__pypackages__`` directory must be inside of ``~/bin/`` directory.
```
